src/models/TMDBConfiguration.ts

```diff
@@ -0,0 +1,26 @@
+import * as mongoose from 'mongoose';
+import { Schema, Document } from 'mongoose';
+
+const THREE_DAYS_IN_SECONDS = 259200;
+
+export interface TMDBConfigurationInterface {
+  imagePrepend: string;
+}
+
+export interface TMDBConfigurationInterfaceDocument extends Document, TMDBConfigurationInterface {}
+
+const TMDBConfigurationSchema: Schema = new Schema({
```

Comment:
What is the purpose for this data? Does this just store one document?

Maybe if this is transient data it could just be stored in memory?

Reply:
The TMDB API exposes this config endpoint to let them change their base URL, and then each image they return is a relative path, so to get an image you combine those two pieces of information. Docs are at https://developers.themoviedb.org/3/configuration/get-api-configuration

The reason I wanted to store it in the db is because they recommend polling every few days for updates. That's why it has this rolling expiry https://github.com/UniversalMediaServer/api/pull/645/files#diff-57f3ed6003187ca5efe94f4a05a18e6d2deeb1c24ecb8be9b6f15136da6cdb8bR16

Reply:
@SubJunk ah yep got it. So how often is the client going to hit this endpoint?

Reply:
I was thinking the same, 3 days